web/angular-wallet/src/app/layout/components/toolbar/toolbar.component.html

```diff
@@ -29,12 +29,15 @@
                     class="user-button">
                 <div fxLayout="row" fxLayoutAlign="center center">
                     <span class="username mr-12" fxHide fxShow.gt-sm>
-                      <ng-container *ngIf="selectedAccount.type === 'offline'">
-                        <mat-icon class="account-type-icon" matTooltip="{{'hint_offline_account' | i18n}}">remove_red_eye</mat-icon>
-                      </ng-container>
-                      <ng-container *ngIf="selectedAccount.type !== 'offline'">
-                        <mat-icon class="account-type-icon" matTooltip="{{'hint_online_account' | i18n}}">verified_user</mat-icon>
-                      </ng-container>
+                        <ng-container *ngIf="selectedAccount.type === 'offline'">
+                            <mat-icon class="account-type-icon" matTooltip="{{'hint_offline_account' | i18n}}">remove_red_eye</mat-icon>
+                        </ng-container>
+                        <ng-container *ngIf="selectedAccount.type !== 'offline' && selectedAccount.confirmed">
+                            <mat-icon class="account-type-icon" matTooltip="{{'hint_online_account' | i18n}}">verified_user</mat-icon>
+                        </ng-container>
+                        <ng-container *ngIf="selectedAccount.type !== 'offline' && !selectedAccount.confirmed">
```

Comment:
ok. hopefully we won't need more status...otherwise it would be better to move the logic into component.ts

Reply:
Agreed... even this is pushing it.